SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/SubJobWebRunner/SubJobWebRunner.swift

```diff
@@ -49,6 +49,14 @@ public protocol SubJobWebRunning: CCFCommunicationDelegate {
              actionsHandler: ActionsHandler?,
              showWebView: Bool) async throws -> ReturnValue
 
+    /// Customization point for a given action when it's expected to run as the next action
+    /// Here we can set the stage, invoke other services, change the retry counts, etc
+    /// By default, the webViewHandler should execute the action
+    ///
+    /// Returns `true` if the action has been executed and we should early return, not passing it to the webViewHandler
+    /// Returns `false` if the action should be passed to the webViewHandler to execute
+    func canPerformNextAction(_ action: Action) async -> Bool
```

Comment:
This follows the same pattern as `optional func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool`